<ui version="4.0" >
 <class>pSearch</class>
 <widget class="pDockWidget" name="pSearch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>162</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Search - Replace</string>
  </property>
  <widget class="QWidget" name="wCentral" >
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QGridLayout" >
      <item row="1" column="0" >
       <widget class="QLabel" name="lSearchText" >
        <property name="text" >
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leSearch" >
        <property name="toolTip" >
         <string>Enter the string or regular expression to find.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2" >
       <widget class="QToolButton" name="tbPrevious" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Search Previous</string>
        </property>
        <property name="text" >
         <string>Previous</string>
        </property>
        <property name="icon" >
         <iconset resource="../../resources/resources.qrc" >:/edit/icons/edit/previous.png</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QToolButton" name="tbNext" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Search Next</string>
        </property>
        <property name="text" >
         <string>Next</string>
        </property>
        <property name="icon" >
         <iconset resource="../../resources/resources.qrc" >:/edit/icons/edit/next.png</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QCheckBox" name="cbCaseSensitive" >
        <property name="toolTip" >
         <string>If it's checked then the search is case sensitive.</string>
        </property>
        <property name="text" >
         <string>Case</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6" >
       <widget class="QCheckBox" name="cbRegExp" >
        <property name="toolTip" >
         <string>If it's checked then expr is interpreted as a regular expression rather than a simple string.</string>
        </property>
        <property name="text" >
         <string>RegExp</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lReplaceText" >
        <property name="text" >
         <string>Replace:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="leReplaceText" >
        <property name="toolTip" >
         <string>Replace the current selection with this string</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QToolButton" name="tbReplace" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Replace</string>
        </property>
        <property name="text" >
         <string>Replace</string>
        </property>
        <property name="icon" >
         <iconset resource="../../resources/resources.qrc" >:/edit/icons/edit/search.png</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2" >
       <widget class="QToolButton" name="tbReplaceAll" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Replace All</string>
        </property>
        <property name="text" >
         <string>Replace All</string>
        </property>
        <property name="icon" >
         <iconset resource="../../resources/resources.qrc" >:/edit/icons/edit/replace.png</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="5" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="lPath" >
          <property name="text" >
           <string>Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePath" >
          <property name="toolTip" >
           <string>Path for searching</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbPath" >
          <property name="toolTip" >
           <string>Choose dirrectory</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="lMask" >
          <property name="text" >
           <string>Mask:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMask" >
          <property name="toolTip" >
           <string>Mask of file names. Use colon for separate masks</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pDockWidget</class>
   <extends>QDockWidget</extends>
   <header>pDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leSearch</tabstop>
  <tabstop>leReplaceText</tabstop>
  <tabstop>cbCaseSensitive</tabstop>
  <tabstop>cbRegExp</tabstop>
  <tabstop>tbPrevious</tabstop>
  <tabstop>tbReplace</tabstop>
  <tabstop>tbNext</tabstop>
  <tabstop>tbReplaceAll</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>leSearch</sender>
   <signal>returnPressed()</signal>
   <receiver>tbNext</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>96</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel" >
     <x>45</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leReplaceText</sender>
   <signal>returnPressed()</signal>
   <receiver>tbReplace</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>95</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel" >
     <x>16</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
